.changeset/yellow-hotels-visit.md

```diff
@@ -0,0 +1,6 @@
+---
+'simple-validators': patch
+'myst-templates': patch
```

Comment:
I think that these are out of date? Can you double check?

Reply:
> Is there a reason we can't use native `new Date(str)` for that?

Yes -- the RFC 2822 handling is not "standard" -- we're building a new rule from the ABNF grammar for RFC2822. That is, it's defined by the grammar in that RFC but we're not using their `date-time` rule. So, we need to roll our own.